packages-ts/starknet-gauntlet/test/utils/network.ts

```diff
@@ -40,24 +40,30 @@ export abstract class IntegratedDevnet {
 
 class VenvDevnet extends IntegratedDevnet {
   private command: string
+  private opts: any
 
-  constructor(port: string) {
+  constructor(port: string, opts: any) {
     super(port)
 
+    this.opts = opts
     this.command = 'starknet-devnet'
   }
 
   protected async spawnChildProcess(): Promise<ChildProcess> {
-    return spawn(this.command, ['--port', this.port, '--gas-price', '1'])
+    let args = ['--port', this.port, '--gas-price', '1']
+    if (this.opts?.seed) {
+      args.push('--seed', this.opts.seed.toString())
+    }
+    return spawn(this.command, args)
```

Comment:
This was an easy fix, tests passed once I added support for a custom seed. We could probably type this opts value better though